sklearn/covariance/tests/test_graphical_lasso.py

```diff
@@ -248,12 +249,71 @@ def test_graphical_lasso_cv_scores():
         X
     )
 
+    _assert_graphical_lasso_cv_scores(
+        cov=cov,
+        n_splits=splits,
+        n_refinements=n_refinements,
+        n_alphas=n_alphas,
+    )
+
+
+# TODO(1.5): remove in 1.5
```

Comment:
this is already removed?

Reply:
I don't think this is removed in main yet.